src/hooks/resources/user.ts

```diff
@@ -40,9 +40,12 @@ sbvrUtils.addPureHook('DELETE', 'resin', 'user', {
 		if (userIdBind == null) {
 			throw new BadRequestError('You must provide user ID');
 		}
+		if (!('bind' in userIdBind)) {
```

Comment:
These checks can be changed to use `isBindReference` from https://github.com/balena-io-modules/odata-to-abstract-sql/pull/66 to make it nicer but it's not a blocker